scripts/travis/prepare-integration.sh

```diff
@@ -38,8 +38,7 @@ chmod 777 logs/*
 # Workaround for Travis on Ubuntu Xenial so local runner integration tests work
 # when executing them under user "stanley" (by default Travis checks out the
 # code and runs tests under a different system user).
-# NOTE: We need to pass "--exe" flag to nosetests when using this workaround.
-if [ "${UBUNTU_VERSION}" == "xenial" ]; then
-  echo "Applying workaround for stanley user permissions issue to /home/travis on Xenial"
-  chmod 777 -R /home/travis
+if [ "${UBUNTU_VERSION}" == "xenial" ] || [ "${UBUNTU_VERSION}" == "bionic" ]; then
```

Comment:
If you duplicated this logic into the new `scripts/travis/permissions-workaround.sh`, then I would remove this code here, and simply call that script here. Alternatively, you could use make magic to build both targets.